build_scripts/debian/build.sh

```diff
@@ -74,7 +56,8 @@ make install
 # It does not affect the built .deb file though.
 $source_dir/python_env/bin/pip3 install wheel
 for d in $source_dir/src/azure-cli $source_dir/src/azure-cli-core $source_dir/src/azure-cli-nspkg $source_dir/src/azure-cli-command_modules-nspkg $source_dir/src/command_modules/azure-cli-*/; do cd $d; $source_dir/python_env/bin/python3 setup.py bdist_wheel -d $tmp_pkg_dir; cd -; done;
```

Comment:
if easy, please move the loop body to new lines. This line is too long

Reply:
Will do in next PR.